js/circle.js

```diff
@@ -1,20 +1,34 @@
-import { Circle, Game, Scene } from 'athenajs';
-
-// const myGame = new Game({
-//     name: 'sample-circle',
-//     showFps: true,
-//     width: 320,
-//     height: 200
-// });
+import { Scene } from 'athenajs';
+import { MyCircle, MySprite, MyFont } from 'objects/my_objects';
 
 // create a new scene
-const myScene = new Scene();
+const myScene = new class objectsScene extends Scene {
+    /**
+     * Load anny need resources here
+     * 
+     */
+    setup() {
+        debugger;
+        this.loadImage('img/bitmapFont.png', 'myFont');
+        this.loadImage('img/sprites.png', 'mySheet');
```

Comment:
```diff
- this.loadImage('img/bitmapFont.png', 'myFont');
- this.loadImage('img/sprites.png', 'mySheet');
+ this.loadImage(require('../img/bitmapFont.png'), 'myFont');
+ this.loadImage(require('../img/sprites.png'), 'mySheet');
```

Use webpack's `url-loader` : https://github.com/topheman/webpack-babel-starter/blob/master/webpack.config.js#L191 , that way, the assets will be part of the bundle when building.